content/hardware/04.pro/boards/portenta-x8/tutorials/wordpress-webserver/content.md

```diff
@@ -115,6 +115,6 @@ In this tutorial we went through how to install and run a Wordpress and database
 
 ## Troubleshooting
 
-- If the containers aren't installing or running correctly check if there are any other containers currently running on the same ports as the ones used by the Wordpress container. You can check this with ``docker ps -a``.
+- If the containers are not installing or running correctly, check if there are any other containers currently running on the same ports as the ones used by the Wordpress container. You can check this with ``docker ps -a``.
 - If there is any issue running docker commands, make sure you are using ``sudo`` before the commands.
-- If you can't connect to the site when everything is running you can double check the X8s ip address. Run the command `ip s a` in the adb shell. This will display the X8s ip address via usb and wifi. Try connecting via those ip addresses if all else fails.
\ No newline at end of file
+- If you cannot connect to the site when everything is running, you can double check the X8s ip address. Run the command `ip s a` in the adb shell. This will display the X8's ip address via usb and WiFi. Try connecting via those ip addresses if all the rest fails.
```

Comment:
```suggestion
- If you cannot connect to the site when everything is running, you can double check the X8s IP address. Run the command `ip s a` in the **adb shell**. This will display the X8's IP address via USB and WiFi. Try connecting via those IP addresses if all the rest fails.
```